opt4j-optimizers/src/main/java/org/opt4j/optimizers/ea/aeseh/package-info.java

```diff
@@ -0,0 +1,8 @@
+/**
+ * Package for the classes of the Adaptive ε-Sampling ε-Hood MOEA.
+ */
+/**
```

Comment:
`*/
/**`
should just be one comment.

See for example org.opt4j.core.package-info.java. Some general information, how all the different classes/interfaces in this package interact would be great to easier get an overview.